src/api/user/service.js

```diff
@@ -35,4 +36,27 @@ export class UserService {
   logout() {
     this.currentUser = null;
   }
+
+  create(username, roles) {
+    return axios.post('backend://users', { username, roles: [roles] })
+      .then((response) => this.users = this.hydrateArray(response.data)); // todo remove this temporary fix.
+  }
+
+  hydrateArray(data) {
+    return data.map(this.hydrate);
+  }
+
+  resolveUsers() {
+    return axios.get('backend://users')
+      .then((response) => this.users = this.hydrateArray(response.data));
```

Comment:
Same as above. but do we need to set this.users here? 
response.data.map(UserEntity.create)